apps/app/repository.tf

```diff
@@ -152,22 +152,73 @@ EOF
 }
 
 resource "github_actions_environment_secret" "acr_name" {
+  count           = local.is_prod ? 0 : 1
   repository      = github_repository.app.name
   environment     = github_repository_environment.app.environment
   secret_name     = "ACR_NAME"
   plaintext_value = data.azurerm_container_registry.serve.name
 }
 
 resource "github_actions_environment_secret" "acr_token_username" {
+  count           = local.is_prod ? 0 : 1
   repository      = github_repository.app.name
   environment     = github_repository_environment.app.environment
   secret_name     = "ACR_USERNAME"
   plaintext_value = azurerm_container_registry_token.app_access.name
 }
 
 resource "github_actions_environment_secret" "acr_token_password" {
+  count           = local.is_prod ? 0 : 1
   repository      = github_repository.app.name
   environment     = github_repository_environment.app.environment
   secret_name     = "ACR_PASSWORD"
   plaintext_value = azurerm_container_registry_token_password.app_access.password1[0].value
 }
+
+resource "github_actions_environment_secret" "sp_client_id" {
+  count           = local.is_prod ? 1 : 0
+  repository      = github_repository.app.name
+  environment     = github_repository_environment.app.environment
+  secret_name     = "ARM_CLIENT_ID"
+  plaintext_value = azuread_application.webapp_sp[0].application_id
+}
+
+resource "github_actions_environment_secret" "sp_client_secret" {
+  count           = local.is_prod ? 1 : 0
+  repository      = github_repository.app.name
+  environment     = github_repository_environment.app.environment
+  secret_name     = "ARM_CLIENT_SECRET"
+  plaintext_value = azuread_application_password.webapp_sp[0].value
+}
+
+resource "github_actions_environment_secret" "tennant_id" {
```

Comment:
Same here - don't think the cli sp login will need the tenant id as the Sp will only be linked to one

Reply:
as [above](https://github.com/UCLH-Foundry/FlowEHR/pull/140#discussion_r1126438353)